docs/how-to/monitor-a-router/README.md

```diff
@@ -0,0 +1,6 @@
+# Monitoring a Router
+
+Turing provides a convenient user interface for monitoring the router deployments and getting the standard metrics out of the box, and configuring alerts based on the performance of the routers. To take advantage of these features, an appropriate monitoring / alerting backend should have been configured.
```

Comment:
super nit: Should we standardize `out of the box` as `out-of-the-box` everywhere in the docs? I noticed `out-of-the-box` being used in `docs/dev-docs/monitoring-and-alerting/configure-monitoring-backend.md`.